roles/kuryr/templates/controller-deployment.yaml.j2

```diff
@@ -9,6 +9,11 @@ metadata:
   namespace: {{ kuryr_namespace }}
 spec:
   replicas: 1
+  strategy:
+    type: RollingUpdate
```

Comment:
Should this be RollingUpdate or simply Rolling?

Reply:
RollingUpdate according to https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#deploymentstrategy-v1-apps